react-native/react/actions/login2.js

```diff
@@ -82,81 +62,27 @@ export function setCodePageOtherDeviceRole (otherDeviceRole) {
   }
 }
 
-let timerId = null
-function resetCountdown () {
-  clearInterval(timerId)
-  timerId = null
-}
-
-// Count down until 0, then make a new code
-function startCodeGenCountdown (mode) {
-  let countDown = Constants.countDownTime
-
-  return function (dispatch) {
-    resetCountdown()
-    timerId = setInterval(() => {
-      countDown -= 1
-
-      if (countDown <= 0) {
-        dispatch(startCodeGen(mode))
-      } else {
-        dispatch({
-          type: Constants.setCountdown,
-          countDown
-        })
-      }
-    }, 1000)
+function generateQRCode (dispatch, getState) {
```

Comment:
Helper to make the qr code. i have a ticket to see if we want go to generate this as there is logic to do it in the terminal